pages/index.js

```diff
@@ -11,18 +13,19 @@ const Home = () => {
     });
     const [formErrors, setFormErrors] = React.useState({});
 
+    const [_, setUserName] = useLocalStorage('username', 'Anonymous');
```

Comment:
If an username is already present in localStorage we could display it by default